### ISSUE TITLE
Synchronise Profiles/data between two browsers/PC

### ISSUE THREAD

**Issue by zakary2841** [dailyscape/dailyscape.github.io#28]
*This is a suggestion/enhancement* 
I use two PCs to play the game. One at home and one on my laptop. I would love to be able to tick some dailies on my laptop then finish the rest on my PC. Or change the list. Etc. etc.

**Comment by 607ch00** [comment]
Hi, that sounds awesome, of course I'd love to do this.  I'm just not 100% sure how to store the data for a specific user without an actual server with a database behind the application.  

I tried to make this app super lightweight and permanently free/available so it's all just HTML and Javascript. Everything else is stored just in your browser itself.

I'm open to suggestions?

**Comment by 607ch00** [comment]
I just remembered I had opened this ticket #7 to think about a way to maybe generate a link that could save all your data and possibly you could just message it to yourself on your other computer?  Does this sound too clunky?

I was initially thinking it would be used more for sharing a preset list of tasks with other people but it could possibly apply to what you're saying.

**Comment by zakary2841** [comment]
I've created a script to copy the Local Storage between PCs on a different browser. So I have managed to get to work (although clunky). At first, I tried to use something like SyncMyCookie to do it until I realised you don't store the info there. 

I guess if your plan is to do everything without a server there's not a good way to do it. It's up to you whether you decide to do something with this. I appreciate the website either way!
